libraries/net/node.cpp

```diff
@@ -69,6 +70,8 @@
 #include <fc/crypto/rand.hpp>
 #include <fc/network/rate_limiting.hpp>
 #include <fc/network/ip.hpp>
+#include <fc/network/resolve.hpp>
+#include <fc/smart_ref_impl.hpp>
```

Comment:
I guess we no longer need to include this.

Reply:
Removed